platform/commonUI/edit/test/actions/SaveAsActionSpec.js

```diff
@@ -156,6 +156,28 @@ define(
                 expect(SaveAsAction.appliesTo(actionContext)).toBe(false);
             });
 
+            it("uses the editor capability to save the object", function () {
+                mockEditorCapability.save.andReturn(new Promise(function () {}));
```

Comment:
Also I noticed that we're using Jasmine 2.3; should I update SaveAsSpec to use the new async testing functionality? http://jasmine.github.io/2.0/introduction.html#section-Asynchronous_Support